stable/inbucket/values.yaml

```diff
@@ -11,13 +11,42 @@ service:
   loadBalancerSourceRanges: []
   type: ClusterIP
   port:
-    http: 10080
-    smtp: 10025
-    pop: 10110
+    http: 9000
+    smtp: 2500
+    pop3: 1100
   nodePort:
     http: ""
     smtp: ""
-    pop: ""
+    pop3: ""
+
+env:
```

Comment:
Please change it to `extraEnv`. That's more in line with what other charts do.

Reply:
done